app/renderer/components/Inspector/SelectedElement.js

```diff
@@ -95,6 +96,18 @@ class SelectedElement extends Component {
       });
     }
 
+    const copyAttributesToClipboard = (dataSource) => {
+      let result = '';
+      Object.entries(dataSource).forEach(function (obj) {
```

Comment:
I'd rather reduce dataSource entries to a json object and then return its stringified representation

Reply:
also, replacing `obj` with `[, smth]` would be more readable

Reply:
or even `[,{key, value}]`

Reply:
Hi @mykola-mokhnach 

The current Array looks like

```
[{"key":"elementId","value":"ebf5e6eb-8b9e-409e-adbd-1c6030d461e4","name":"elementId"},{"key":"index","value":"0","name":"index"},{"key":"text","value":"","name":"text"},{"key":"class","value":"android.view.ViewGroup","name":"class"},{"key":"package","value":"com.dozuki.ifixit","name":"package"},{"key":"content-desc","value":"","name":"content-desc"},{"key":"checkable","value":"false","name":"checkable"},{"key":"checked","value":"false","name":"checked"},{"key":"clickable","value":"false","name":"clickable"},{"key":"enabled","value":"true","name":"enabled"},{"key":"focusable","value":"false","name":"focusable"},{"key":"focused","value":"false","name":"focused"},{"key":"scrollable","value":"false","name":"scrollable"},{"key":"long-clickable","value":"false","name":"long-clickable"},{"key":"password","value":"false","name":"password"},{"key":"selected","value":"false","name":"selected"},{"key":"bounds","value":"[0,0][1080,1920]","name":"bounds"},{"key":"resource-id","value":"android:id/decor_content_parent","name":"resource-id"},{"key":"instance","value":"0","name":"instance"}]
```
and I will keep the event of the button likes

```
 <Tooltip title={t('Copy Attributes to Clipboard')}>
              <Button
                disabled={!elementId}
                id='btnCopyAttributes' icon="copy"
                onClick={() => clipboard.writeText(JSON.stringify(dataSource))}/>
            </Tooltip>
```

Does it make sense?